Client/multiplayer_sa/multiplayer_shotsync.cpp

```diff
@@ -71,18 +71,23 @@ extern DeathHandler*          m_pDeathHandler;
 extern FireHandler*           m_pFireHandler;
 extern ProjectileHandler*     m_pProjectileHandler;
 extern ProjectileStopHandler* m_pProjectileStopHandler;
+extern TakeAStepHandler*      m_pTakeAStepHandler;
 
 char szDebug[255] = {'\0'};
 
 DWORD   RETURN_CProjectile__AddProjectile = 0x401C3D;
 DWORD   RETURN_CProjectile__CProjectile = 0x4037B3;
+DWORD   CONTINUE_CPed_DoFootLanded = 0x5E5386;

```

Comment:
Prefix your variable with `static const` to keep the value in this translation unit and constant.